docker/docker-compose-with-ccd.yml

```diff
@@ -3,61 +3,65 @@ networks:
   default: {}
 services:
   send-letter-service:
-    build:
-      context: .
-      args:
-      - http_proxy
-      - https_proxy
-      - no_proxy
-    image: docker.artifactory.reform.hmcts.net/reform/send-letter-service
+    image: hmcts.azurecr.io/hmcts/rpe-send-letter-service:latest
     container_name: send-letter-service
     environment:
-    - JAVA_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8003
     - APPINSIGHTS_INSTRUMENTATIONKEY=DUMMY
     - SERVER_PORT=8485
-    - FTP_FINGERPRINT="fingerprint"
-    - FTP_HOSTNAME="hostname"
+    - FTP_FINGERPRINT=fingerprint
+    - FTP_HOSTNAME=hostname
     - FTP_PORT=22
-    - FTP_PRIVATE_KEY="private"
-    - FTP_PUBLIC_KEY="public"
-    - FTP_REPORTS_FOLDER="/reports/"
-    - FTP_TARGET_FOLDER="/target/"
-    - FTP_USER="user"
+    - FTP_PRIVATE_KEY=private
+    - FTP_PUBLIC_KEY=public
+    - FTP_REPORTS_FOLDER=/reports/
+    - FTP_TARGET_FOLDER=/target/
+    - FTP_USER=user
     - LETTER_TRACKING_DB_PASSWORD=password
-    - S2S_URL="false"
+    - LETTER_TRACKING_DB_USER=letterservice
+    - S2S_URL=false
     - SCHEDULING_ENABLED
     - SPRING_FLYWAY_ENABLED=false
     - ENCRYPTION_ENABLED
     - ENCRYPTION_PUBLIC_KEY
-    - LETTER_TRACKING_DB_HOST=localhost
     # these environment variables are used by java-logging library
     - ROOT_APPENDER
     - JSON_CONSOLE_PRETTY_PRINT
     - ROOT_LOGGING_LEVEL
     - REFORM_SERVICE_NAME
     - REFORM_TEAM
     - REFORM_ENVIRONMENT
+    - LETTER_TRACKING_DB_HOST=send-letter-database
     ports:
-    - 8003:8003
     - 8485:8485
     links:
     - send-letter-database
     depends_on:
-    - send-letter-database
+      send-letter-database:
+        condition: service_healthy
 
   send-letter-database:
-    image: docker.artifactory.reform.hmcts.net/reform/send-letter-database
+    image: postgres:9.6-alpine
```

Comment:
try to use same `shared-database:` if possible.
I think you can create table in it.

Reply:
If you mean the init0-db.sh then no. Could not find a way to make it work as the bulk print db needs specific configs ( ROLE, PASSWORD, DATABASE) are all different.

Reply:
Will try to create the database from there.

Reply:
Also can't create table from the script as the db needs to exist

Reply:
I agree with Praveen, we should reuse `shared-database` container. It already has 4 CCD and 1 IDAM database, I can't see why we can't have `send-letter` database there.

Reply:
It's now using the shared-database. Thanks